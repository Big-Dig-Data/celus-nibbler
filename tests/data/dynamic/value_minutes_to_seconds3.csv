Month,Mins,Users
"2020-01","0.1","1"
"2020-02","0.5","3"
"2020-03","3.1","5"
"Total","3.7","8"
