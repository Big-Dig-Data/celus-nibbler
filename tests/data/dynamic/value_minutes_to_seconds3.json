{
  "kind": "non_counter.metric_based",
  "version": 1,
  "parser_name": "value_minutes_to_seconds3",
  "data_format": {
    "name":"minutes_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [
      ["Mins", "Seconds"]
  ],
  "metric_value_extraction_overrides": {
    "Seconds": "minutes_to_seconds"
  },
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "^Month$",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 1
                    },
                    "direction": "right"
                },
                "role": "metric"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 1
          },
          "direction": "right"
        }
      },
      "dates": {
          "source": {
              "coord": {
                  "row": 1,
                  "col": 0
              },
              "direction": "down"
          },
          "extract_params": {
              "on_validation_error": "stop"
          },
          "role": "date"
      },
      "titles": null,
      "title_ids": [],
      "dimensions": [],
      "kind": "non_counter.metric_based"
    }
  ]
}
