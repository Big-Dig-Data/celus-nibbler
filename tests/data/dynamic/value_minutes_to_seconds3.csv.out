"2020-01-01","2020-01-31","","","","","","","Users","1"
"2020-02-01","2020-02-29","","","","","","","Users","3"
"2020-03-01","2020-03-31","","","","","","","Users","5"
"2020-01-01","2020-01-31","","","","","","","Seconds","6"
"2020-02-01","2020-02-29","","","","","","","Seconds","30"
"2020-03-01","2020-03-31","","","","","","","Seconds","186"
